springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/types/channel/operation/message/Message.java

```diff
@@ -20,8 +20,19 @@
 @AllArgsConstructor
 public class Message {
 
+    public static final String DEFAULT_SCHEMA_FORMAT = "application/vnd.oai.openapi+json;version=3.0.0";
+
     @Builder.Default
-    private String schemaFormat = "application/vnd.oai.openapi+json;version=3.0.0";
+    private String schemaFormat = DEFAULT_SCHEMA_FORMAT;
+
+    /**
+     * Unique string used to identify the message.
+     * <p>
+     * The id MUST be unique among all messages described in the API. The messageId value is case-sensitive.
+     * Tools and libraries MAY use the messageId to uniquely identify a message, therefore, it is RECOMMENDED to
+     * follow common programming naming conventions.
+     */
+    private String messageId;
```

Comment:
This field is new and can be `@Nullable` -> disregard. I guess we are not applying `@Nullable` annotations for this class.

Or do we want to provide a sensible, opinionated default (then also for all scanners) - in a different PR?

Reply:
None of the fields in the class are using the `@Nullable` annotation, even when some of them are also optional. This class is also build via a Builder, so there's really nothing that should be breaking.

I'm a bit hesitant to provide a default value. The only element that we could use as really unique would be the full class name. But that seems like a poor id, since it's language related.

Reply:
Fair enough. I recently came across a code generator, which requires a `messageId` (https://github.com/asyncapi/java-spring-template)

But that is beyond the scope of this PR and even if, we can stick to the current default of null. Providing a default value could possibly be part of an (opt-in) customizer.